vortex-vector/src/fixed_size_list/scalar.rs

```diff
@@ -84,9 +86,46 @@ impl ScalarOps for FixedSizeListScalar {
         }
     }
 
-    fn repeat(&self, _n: usize) -> VectorMut {
-        // TODO(ngates): add "repeat(n)" to the vector ops trait
-        todo!()
+    fn repeat(&self, n: usize) -> VectorMut {
+        if n == 0 {
+            // Return an empty vector with the correct structure
+            let list_size = self.0.list_size();
+            let scalar_elements = self.0.elements();
+            let elements = scalar_elements.slice(0..0).into_mut();
+            let validity = MaskMut::new_true(0);
+            return unsafe {
+                VectorMut::FixedSizeList(FixedSizeListVectorMut::new_unchecked(
+                    Box::new(elements),
+                    list_size,
+                    validity,
+                ))
+            };
+        }
+
+        let list_size = self.0.list_size();
+
+        // Get the elements from the scalar's inner length-1 vector and repeat them
+        // Clone the inner Vector from the Arc
+        let scalar_elements = self.0.elements();
+        let mut elements = scalar_elements.as_ref().clone().into_mut();
+        elements.reserve((n - 1) * list_size as usize);
+
+        // Repeat the elements n-1 more times (we already have 1 copy)
+        for _ in 1..n {
+            elements.extend_from_vector(scalar_elements.as_ref());
+        }
+
+        // Build validity for n lists
+        let validity = MaskMut::new(n, self.is_valid());
```

Comment:
If is_valid, we should just append_n zeroes to the elements, faster than extend_from_vector